presto-hive/src/main/java/com/facebook/presto/hive/security/SqlStandardAccessControl.java

```diff
@@ -392,10 +390,15 @@ private boolean isDatabaseOwner(ConnectorTransactionHandle transaction, Identity
 
     private boolean isTableOwner(ConnectorTransactionHandle transaction, Identity identity, SchemaTableName tableName)
     {
-        return checkTablePermission(transaction, identity, tableName, OWNERSHIP);
+        return checkTablePermission(transaction, identity, tableName, OWNERSHIP, false);
     }
 
-    private boolean checkTablePermission(ConnectorTransactionHandle transaction, Identity identity, SchemaTableName tableName, HivePrivilege requiredPrivilege)
+    private boolean checkTablePermission(
+            ConnectorTransactionHandle transaction,
+            Identity identity,
+            SchemaTableName tableName,
+            HivePrivilege requiredPrivilege,
+            boolean grantOptionRequired)
```

Comment:
Can we combine the two arguments -- `HivePrivilege requiredPrivilege` and `boolean grantOptionRequired` into an object of type `HivePrivilegeInfo` and pass that instead?